From PeterHuewe@gmx.de  Thu Jan 14 17:00:10 2010
From: Peter Huewe <PeterHuewe@gmx.de>
Date: Thu, 7 Jan 2010 21:36:38 +0100
Subject: Staging: wlan-ng: Add select WEXT_PRIV to Kconfig to prevent build failure
To: "Greg Kroah-Hartman" <gregkh@suse.de>, Simon Horman <horms@verge.net.au>
Cc: devel@driverdev.osuosl.org, linux-kernel@vger.kernel.org
Message-ID: <201001072136.38318.PeterHuewe@gmx.de>


From: Peter Huewe <peterhuewe@gmx.de>
Date: Thu, 7 Jan 2010 21:21:35 +0100

Without WEXT_PRIV set the p80211wext.c fails to build due to unknown fields in
the iw_handler_def struct.
Those fields are enclosed in WEXT_PRIV conditionals in the prototype
of iw_handler_def in include/net/iw_handler.h

Signed-off-by: Peter Huewe <peterhuewe@gmx.de>
Acked-by: Simon Horman <horms@verge.net.au>
Cc: stable <stable@kernel.org>  [2.6.33 only]
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/wlan-ng/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/wlan-ng/Kconfig
+++ b/drivers/staging/wlan-ng/Kconfig
@@ -2,6 +2,7 @@ config PRISM2_USB
 	tristate "Prism2.5/3 USB driver"
 	depends on WLAN && USB
 	select WIRELESS_EXT
+	select WEXT_PRIV
 	default n
 	---help---
 	  This is the wlan-ng prism 2.5/3 USB driver for a wide range of
